docs/library/pyb.Timer.rst

```diff
@@ -107,6 +107,12 @@ Methods
        measures ticks of ``source_freq`` divided by ``div`` clock ticks.
        ``deadtime`` is only available on timers 1 and 8.
 
+     - ``brk`` - specifies if the break mode is used to kill the output of
+        the PWM when the ``BRK_IN`` input is asserted. The polarity set how the
```

Comment:
would read better something like: "... input is asserted.  The value of this argument determines if break is enabled and what the polarity is, and can be one of ``BRK_OFF``, ``BRK_LOW`` or ``BRK_HIGH``.

I understand you copied this from the code, but the docs should be the source of truth, and the comments in the code eventually removed.